src/dataEntryApp/views/subjectDashBoard/components/subjectDashboardProgramView.js

```diff
@@ -0,0 +1,266 @@
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ListItemText from "@material-ui/core/ListItemText";
+import moment from "moment/moment";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableRow from "@material-ui/core/TableRow";
+import ErrorIcon from "@material-ui/icons/Error";
+
+import Button from "@material-ui/core/Button";
+
+const useStyles = makeStyles(theme => ({
+  enrollButtonStyle: {
+    marginBottom: theme.spacing(2),
+    marginRight: theme.spacing(2)
+  },
+  root: {
+    flexGrow: 1,
+    padding: theme.spacing(2)
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: "left"
+  },
+  programStatusStyle: {
+    color: "red",
+    backgroundColor: "#FFB6C1",
+    borderRadius: "5px"
+  },
+  expansionHeading: {
+    fontSize: theme.typography.pxToRem(15),
+    flexBasis: "33.33%",
+    flexShrink: 0
+  },
+  listItem: {
+    paddingBottom: "0px",
+    paddingTop: "0px"
+  },
+  table: {
+    border: "1px solid rgba(224, 224, 224, 1)"
+  },
+  abnormalColor: {
+    color: "#ff4f33"
+  }
+}));
+
+const ProgramView = ({ programData }) => {
+  const classes = useStyles();
+  const [expanded, setExpanded] = React.useState(false);
+
+  const handleChange = panel => (event, isExpanded) => {
+    setExpanded(isExpanded ? panel : false);
+  };
+
+  return (
+    <div>
+      <Grid container spacing={3}>
+        <Grid item xs={6}>
+          <label>{programData.operationalProgramName} program details</label>
+        </Grid>
+        <Grid item xs={6}>
+          <Fab
+            className={classes.enrollButtonStyle}
+            variant="extended"
+            color="primary"
+            aria-label="add"
+          >
+            Growth Chart
+          </Fab>
+          <Fab
+            className={classes.enrollButtonStyle}
+            variant="extended"
+            color="primary"
+            aria-label="add"
+          >
+            Vaccinations
+          </Fab>
+          <Fab
+            className={classes.enrollButtonStyle}
+            variant="extended"
+            color="primary"
+            aria-label="add"
+          >
+            New Program Visit
+          </Fab>
+        </Grid>
+      </Grid>
+      <Paper className={classes.root}>
+        <ExpansionPanel expanded={expanded === "panel1"} onChange={handleChange("panel1")}>
+          <ExpansionPanelSummary
+            expandIcon={<ExpandMoreIcon />}
+            aria-controls="panel1bh-content"
+            id="panel1bh-header"
+          >
+            <Typography className={classes.expansionHeading}>Enrollment details</Typography>
+          </ExpansionPanelSummary>
+          <ExpansionPanelDetails>
+            <Grid item xs={12}>
+              <List>
+                {programData
+                  ? programData.observations.map(element => {
```

Comment:
@RashmiDavandePsl 
observations rendering code is duplicated in the profile tab and program tab. Extract it out as a component and use it.